ledfx/effects/__init__.py

```diff
@@ -150,6 +150,8 @@ def smooth(x, sigma):
 
     if len(x) == 0:
         raise ValueError("Cannot smooth an empty array")
+    if sigma == 0:
+        raise ValueError("Sigma cannot be zero")
```

Comment:
I'm happy with the assert here, that's a good catch.

Reply:
Actually, I wonder if we can be even smarter here - if sigma is 0, just return the array as is, since a call to smooth with 0 just means we want the array back as is?